sdk/build.sh

```diff
@@ -0,0 +1 @@
+npm run build
```

Comment:
there is no sense to define this kind of script IMO. It will be better to move commands from `tests.sh`:
```
npm i
npm run lint
npm run wasm:build
npm run build
```
and later:
```
# Build and test SDK
cd sdk
./build.sh
./tests.sh
```